src/Library/demos/Level Bars/main.blp

```diff
@@ -0,0 +1,130 @@
+using Gtk 4.0;
+using Adw 1;
+Adw.Clamp{
+  Box {
+    orientation: vertical;
+    margin-top: 24;
+
+    Label {
+      label: "Level Bars";
+      margin-top: 6;
+      margin-bottom: 6;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: _("A bar widget used as a level indicator");
+      margin-top: 6;
+      margin-bottom: 24;
+    }
+
+    Box {
+      orientation: vertical;
+      vexpand: true;
+      spacing: 18;
+      margin-start: 18;
+      margin-end: 18;
+
+      Label {
+        label: _("Continuous Level Bar");
+      }
+
+      Box{
+        spacing:18;
+        halign: start;
+
+        Label {
+          label: _("Current Battery Level");
+        }
+
+        SpinButton spin_button{
+          digits:1;
+          adjustment: Gtk.Adjustment {
+            lower: 0;
+            upper: 100;
+            value:100;
+            step-increment: 5;
+          };
+        }
+      }
```

Comment:
I don't think we need this

it makes the UI harder to read and changing the value of the level is just one property away 

wdyt?

Reply:
please don't mark conversation as resolved if you didn't reply or address the comment

Reply:
Apologies – I forgot to pull, it's all good.